docs/jade-getting-started.md

```diff
@@ -0,0 +1,221 @@
+  * https://github.com/join
+  * https://hub.docker.com/signup
+
+## 2. Connect accounts
+
+> Make sure 2-factor authentication (2FA) is activated on your
+[Broad](https://broad.io/2fa) and [GitHub](https://github.com/settings/security)
+account before starting this process!
+
+Connect your GitHub account to your Broad profile:
+
+1. Go to [Broad people](https://people.broadinstitute.org/me) and select the
+**My Profile** tab.
+2. Link your profile to GitHub by clicking under **Other Profiles**.
+3. [Check if the account is successfully linked](https://github.broadinstitute.org/).
+4. Open each of the following GitHub groups and **Request to join** by going to 
+the Members tab: [Broad Institute Read](https://github.com/orgs/broadinstitute/teams/broad-institute-read),
+[Prometheus](https://github.com/orgs/broadinstitute/teams/prometheus),
+[DSDE Engineering](https://github.com/orgs/broadinstitute/teams/dsde-engineering)
+5. To avoid being overwhelmed with notifications, [add your Broad email address](https://github.com/settings/emails),
+[route the notifications](https://github.com/settings/notifications) to that
+email, and [unfollow projects](https://github.com/watching) that are not
+relevant to your team.
+
+Connect your Docker Hub account to your Broad profile by contacting the DevOps
+team.
+
+## 3. Install Homebrew
+
+[Homebrew](https://brew.sh/) is a [package manager](https://en.wikipedia.org/wiki/Package_manager)
+which enables the installation of software using a single, convenient command
+line interface:
+
+```
+/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+```
+
+Once Homebrew is installed, there are a number of useful development tools that
+should be installed.
+
+1. [Git](https://git-scm.com/) is a version control tool for tracking changes in
+projects and code:
+
+```
+brew install git
+```
+
+2. [jq](https://stedolan.github.io/jq/) is a command line JSON processing tool:
+
+```
+brew install jq
+```
+
+3. [Docker](https://www.docker.com/) is a tool to deliver software in packages
+called containers. Docker for MacOS also includes [Kubernetes](https://kubernetes.io/),
+which deploys groups of containers together in clusters. Once installed, you'll
+need to run Docker once from your list of Applications:
+
+```
+brew cask install docker
+open -a Docker
+```
+
+4. [Helm](https://helm.sh/) streamlines the process of defining, installing, and
+upgrading Kubernetes deployments, which are otherwise challenging to manage:
+
+```
+brew install helm
+helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+helm repo add datarepo-helm https://broadinstitute.github.io/datarepo-helm
+helm plugin install https://github.com/thomastaylor312/helm-namespace
+helm repo update
+```
+
+5. [Skaffold](https://github.com/GoogleContainerTools/skaffold) facilitates the
+continuous development of Kubernetes resources:
+
+```
+brew install skaffold
```

Comment:
This isn't correct at this moment, is it?
@thathert say: "If you brew install you will get the bad version, so you’ll want an older version for now until I get to the bottom of this. "
And: "This version: https://github.com/GoogleContainerTools/skaffold/releases/tag/v1.3.1"

Reply:
I think you can specify the version with Brew too, let me check if that's the case.

Reply:
It's non-trivial but possible to install older versions. You need to identify the Homebrew commit corresponding to an older version of the software package (using `git log <file>`) and then `brew install` the file at that commit.